packages/cli/src/commands/init/init.ts

```diff
@@ -175,13 +176,31 @@ async function installDependencies({
   loader.succeed();
 }
 
+function checkVersionAndTemplate(options: Options, version: string): string {
+  const isTypescriptTemplate = options.template?.includes(
```

Comment:
Make it strict, not includes. We don't want to target other templates by accident